src/index.ts

```diff
@@ -70,10 +70,11 @@ const COMMANDS: { [key: string]: [(...args) => void, string, string] } = {
   'account-id': [c.getAccountId, '', 'get the client account id (indexed DB name)'],
   'support-id': [c.getSupportId, '', 'get the client support id'],
   'version': [ c.getVersion, '', 'get console and relayer versions'],
-  'gift-card-generate':[c.generateGiftCards,'<alias> <quantity> <balance> <token>','generate gift cards'],
-  'give':[c.genBurnerAddress, '<amount>', 'creates a single burner wallet with specified amount and returns redemption url and qr code'],
-  'gift-card-balance': [c.giftCardBalance, '<spending_key> [birthindex]', 'retrieve gift card balance'],
-  'gift-card-redeem': [c.redeemGiftCard, '<spending_key> [birthindex]', 'redeem gift card to the current account'],
+  'gift-card-generate':[c.generateGiftCardLocal, '<amount>', 'creates a single burner wallet with specified amount and returns redemption url and qr code'],
```

Comment:
It seems that you've forgotten to update readme.

Reply:
Sure, thanks for that point!
Fixed in https://github.com/zkBob/zkbob-console/pull/71/commits/f9696ddfbcd31a095acfb5348da9b328cf2033ca